src/bundle/upload.ts

```diff
@@ -301,223 +302,248 @@ async function uploadBundleToCapgoCloud(supabase: SupabaseType, appid: string, b
   spinner.start(`Uploading Bundle`)
   const startTime = performance.now()
 
-  try {
-    if (options.multipart !== undefined && options.multipart) {
-      p.log.info(`Uploading bundle as multipart`)
```

Comment:
Why all this code is deleted ?

Reply:
They aren't deleted. They moved to down in the changes :)